src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -917,6 +919,7 @@ export default class FilingHistoryList extends Mixins(
 
 .v-icon.mdi-gavel {
   color: $gray9;
+  padding-right: 4px;
```

Comment:
You could use class `pr-1` instead of this. (It's easier to maintain if layout stuff is in the template itself.)

Or, you could try `<v-icon left...`.

Reply:
Done